### PR TITLE
Remove duplication in docs

### PR DESCRIPTION
**Pull request by jpm-canonical**
## Summary

Removed usage instructions from readme and rather link to documentation pages.

### CODE REVIEW

File: README.md
Comment (farshidtz) [inline README.md:1]: Add connection command need to stay for installation of an unsigned snap.

File: README.md [hunk 1]
```diff
@@ -7,85 +8,72 @@ The snap packaging makes it easy to run the Chip Tool on Linux distributions.
...
-
--   `110` is the node id being assigned to the app
--   `20202021` is the pin code set on the app
+### Connect interfaces
 
-### Commissioning into Thread network over BLE
-This depends on an OpenThread Border Router (OTBR) with an active network.
-For guidance on setting that up using the [OTBR snap](https://snapcraft.io/openthread-border-router), refer to [this tutorial](https://github.com/canonical/openthread-border-router-snap/wiki/Commission-and-control-a-Matter-Thread-device-via-the-OTBR-Snap).
+When the snap is installed from the Snap Store, the `avahi-observer` and `bluez` interfaces are connected automatically.
+If you install the snap from a local file, like when you built it yourself, you need to connect these interfaces manually:
```
Comment (farshidtz) [inline README.md:40]: You can download the snap from the store and install it from the local file. Once you acknowledge the assertion, snap can be installed without using dangerous mode and the interfaces auto connect:
```console
$ snap download chip-tool
Fetching snap "chip-tool"
Fetching assertions for "chip-tool"
Install the snap with:
   snap ack chip-tool_96.assert
   snap install chip-tool_96.snap

$ snap ack chip-tool_96.assert 

$ sudo snap install ./chip-tool_96.snap 
chip-tool v1.3.0.0+snap from Canonical IoT Labs installed

$ snap connections chip-tool
Interface        Plug                       Slot            Notes
avahi-observe    chip-tool:avahi-observe    :avahi-observe  -
bluez            chip-tool:bluez            :bluez          -
network          chip-tool:network          :network        -
network-bind     chip-tool:network-bind     :network-bind   -
process-control  chip-tool:process-control  -               -
```

So: 
```suggestion
If you install the locally built snap, you need to connect these interfaces manually:
```

File: README.md [hunk 2]
```diff
@@ -7,85 +8,72 @@ The snap packaging makes it easy to run the Chip Tool on Linux distributions.
...
-Discover and pair:
+On **Ubuntu Core**, the `avahi-observe` and `bluez` interfaces are not provided by the system.
+These interfaces are provided by the [Avahi](https://snapcraft.io/avahi) and [BlueZ](https://snapcraft.io/bluez) snaps.
+To install the snaps and connect the interfaces, run:
+
 ```bash
-chip-tool pairing ble-thread 110 hex:0e08...f7f8 20202021 3840
+sudo snap install avahi bluez
+sudo snap connect chip-tool:avahi-observe avahi:avahi-observe
+sudo snap connect chip-tool:bluez bluez:service
```
Comment (farshidtz) [inline README.md:54]: The UC related instructions can also be removed as exist on the documentation (and in in better shape).

**Review by farshidtz — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (farshidtz) [reply README.md:54]: Ignore this. This is more about the different manual connection comments.

Comment (jpm-canonical) [reply README.md:54]: A little unsure about this. Looking at the docs https://canonical-matter.readthedocs-hosted.com/en/latest/how-to/chip-tool-commission-and-control/ it just says to install the avahi and bluez snaps, but not the commands to connect the interfaces.

**Review by farshidtz — APPROVED** [review]

**Pull request merged**